### ISSUE TITLE
Some questions about SGCls Predictions

### ISSUE THREAD

**Issue by ZhuYun97** [KaihuaTang/Scene-Graph-Benchmark.pytorch#43]
## ❓ Questions and Help
SGCls: given bounding boxes and image, predicting labels and relationships.  
It means the model needs to predict labels about bounding boxes. I print the predictions of the SGCls model, but the _pred_scores_ are all 1. I am confused about why these values are not ranging from 0 to 1. Could you help me to understand it?
![image](https://user-images.githubusercontent.com/37068560/84990766-387b0680-b178-11ea-94ae-aef8af643b3a.png)

**Comment by KaihuaTang** [comment]
I think either you are running PredCls by mistake or this image is just very simple with 100% accuracy and very high confidence, because  "labels"(ground-truth) and "pred_labels" are exactly the same.

**Comment by KaihuaTang** [comment]
make sure you train the model with "MODEL.ROI_RELATION_HEAD.USE_GT_BOX True MODEL.ROI_RELATION_HEAD.USE_GT_OBJECT_LABEL False" for SGCls

**Comment by ZhuYun97** [comment]
Thanks, I will test it again.

**Comment by ZhuYun97** [comment]
It seems that I set `MODEL.ROI_RELATION_HEAD.USE_GT_OBJECT_LABEL True`  for SGCls, so I get all 1 scores. Thanks for your help.
